core/src/main/scala/com/metamx/tranquility/config/ConfigHelper.scala

```diff
@@ -0,0 +1,114 @@
+package com.metamx.tranquility.config
+
```

Comment:
Optimize imports / apply codestyle (this file looks like it is a little bit off on each of those)